physcraper/__init__.py

```diff
@@ -1269,24 +1242,21 @@ class IdDicts(object):
                * self.otu_rank: for remote blast to store the rank information
     """
 
-    def __init__(self, config_obj, workdir, mrca=None):
+    def __init__(self, config_obj, workdir):
         """Generates a series of name disambiguation dicts"""
         self.config = config_obj
         assert self.config.email
-        self.ott_to_ncbi = {}  # currently only used to find mcra ncbi id from mrca ott_id
+        self.ott_to_ncbi = {} 
         self.ncbi_to_ott = {}  # used to get ott_id for new Genbank query taxa
         self.ott_to_name = {}  # used in add_otu to get name from otuId
         self.acc_ncbi_dict = {}  # filled by ncbi_parser (by subprocess in earlier versions of the code).
         self.spn_to_ncbiid = {}  # spn to ncbi_id, it's only fed by the ncbi_data_parser, but makes it faster
-        self.ncbiid_to_spn = {}
-        self.mrca_ott = mrca  # mrca_list
-        assert type(self.mrca_ott) in [int, list] or self.mrca_ott is None
-        self.mrca_ncbi = set()  # corresponding ids for mrca_ott list
+        self.ncbiid_to_spn = {} #TODO when is this generated?
```

Comment:
It is generated here. We are feeding in, similar to the two dictionaries above, information from ncbi.

Reply:
:+1: